docs/getting-started.md

```diff
@@ -107,36 +114,37 @@ You can test everything is working by running `ngrok -h` which should print the
     OPTIONS:
       -h, --help      help for ngrok
       -v, --version   version for ngrok
-
+```
 ### Step 3: Connect your agent to your ngrok account
 
 Now that the ngrok agent is installed, let's connect it to your ngrok Account. If you haven't already, [sign up (or log in)](https://dashboard.ngrok.com) to the ngrok Dashboard and get your [Authtoken](https://dashboard.ngrok.com/get-started/your-authtoken). The ngrok agent uses the authtoken (sometimes called tunnel credential) to log into your account when you start a tunnel.
 
 Copy the value and run this command to add the authtoken in your terminal.
-
-    ngrok config add-authtoken TOKEN
+```
+ngrok config add-authtoken TOKEN
+```    
 
 ### Step 4: Start ngrok
 
 Start ngrok by running the following command.
-
-    ngrok http 8000
-
+```
+ngrok http 8000
+```
```

Comment:
Try and add the language to all the code blocks that need it (this is required for syntax highlighting).

Example

```bash

Reply:
@sudobinbash do your prefer ```bash, for some reason it was add random splashes of purple